src/createIsolation.tsx

```diff
@@ -36,29 +36,26 @@ export function createIsolation() {
     );
   };
 
-  const useAtom = ((anAtom: any, options?: any) => {
+  const useStore = () => {
```

Comment:
Do we not want to keep the original signature of useStore?

Reply:
fixed